src/demo/animation.rs

```diff
@@ -4,12 +4,12 @@
 //! - [Sprite animation](https://github.com/bevyengine/bevy/blob/latest/examples/2d/sprite_animation.rs)
 //! - [Timers](https://github.com/bevyengine/bevy/blob/latest/examples/time/timers.rs)
 
-use std::time::Duration;
-
 use bevy::prelude::*;
+use rand::seq::SliceRandom;
+use std::time::Duration;
 
 use super::movement::MovementController;
-use crate::{assets::SfxHandles, audio::sfx::SfxCommands as _, AppSet};
+use crate::{audio::SoundEffect, demo::player::PlayerAssets, screens::Screen, AppSet};
```

Comment:
I'm personally not a fan of `use super::`, but we should be consistent in its usage anyways. Move `crate::demo::player::PlayerAssets` to `super::player::PlayerAssets`.

(GitHub UI won't let me write a `suggestion` for this)

Reply:
I don't like super for this either. Pretty sure existing usages were placed there by someone’s IDE and we should replace them in a follow-up PR instead

Reply:
If we want to replace `super` with `crate` in a separate PR, I'm fine with leaving this as is in this PR.